test/unit/module/real/core/remdiv.cpp

```diff
@@ -50,8 +50,8 @@ EVE_TEST( "Check behavior of remdiv on wide"
   using eve::toward_zero;
   a1 = eve::if_else(eve::is_eqz(a1), eve::one, a1);
   auto [r, d] = eve::remdiv(a0, a1);
-  TTS_ULP_EQUAL( r, map([](auto e, auto f) { return eve::rem(e, f); }, a0, a1), 30);//fma not avail scalar double it seems
-  TTS_ULP_EQUAL( d, map([](auto e, auto f) { return toward_zero(eve::div)(e, f); }, a0, a1), 30);//fma not avail scalar double it seems
+  TTS_ULP_EQUAL( r, map([](auto e, auto f) { return eve::rem(e, f); }, a0, a1), 32);//fma not avail scalar double it seems
+  TTS_ULP_EQUAL( d, map([](auto e, auto f) { return toward_zero(eve::div)(e, f); }, a0, a1), 32);//fma not avail scalar double it seems
```

Comment:
And these?

Reply:
as the size of the vector changed to includes the new aggregate size for avx512, it appears that one value actually trigger a slightly larger ULP.